qiskit_ionq/helpers.py

```diff
@@ -515,17 +517,19 @@ def resolve_credentials(token: str | None = None, url: str | None = None):
     Returns:
         dict[str]: A dict with "token" and "url" keys, for use by a client.
     """
+    # Cache dotenv and environment variable lookups
+    env_values = dotenv_values()
```

Comment:
This will mean that if one calls `resolve_credentials` and doesn't have the env vars set, then sets it, then calls `resolve_credentials` again, which will return the incorrect thing.

No need to cache such a cheap operation IMHO.

Reply:
good point - ill remove caching